psalm.xml.dist

```diff
@@ -4,6 +4,8 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+    findUnusedBaselineEntry="true"
+    findUnusedCode="false"
```

Comment:
We should most likely use `true` here, even if it expands the baseline

Reply:
I'll update to do that.

Reply:
Updated. :)